### PR TITLE
Docs: Steps for adding new committers/TSCers

### PR DESCRIPTION
**Pull request by nzakas**
**What is the purpose of this pull request? (put an "X" next to item)**

[x] Documentation update

**Please check each item to ensure your pull request is ready:**

- [x] I've read the [pull request guide](http://eslint.org/docs/developer-guide/contributing/pull-requests)
- [ ] I've included tests for my change
- [x] I've updated documentation for my change (if appropriate)

**What changes did you make? (Give an overview)**

Listed the steps for adding a new committer or TSC member after they have been approved.

**Is there anything you'd like reviewers to focus on?**

Nothing in particular.

### CODE REVIEW

File: docs/maintainer-guide/governance.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ It is important to recognize that committership is a privilege, not a right. Tha
 
 A Committer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become a TSC member, described below.
 
+#### Process for Adding Committers
+
+1. Add the GitHub user to the "ESLint Team" team
+2. Send welcome email with link to maintainer guide
+3. Add committer email to the ESLint team mailing list
+4. Tweet congratulations to the new committer from the ESLint Twitter account
```
Comment (platinumazure) [inline docs/maintainer-guide/governance.md]: Don't know if we want to mention updating the README as well?

File: docs/maintainer-guide/governance.md [hunk 2]
```diff
@@ -81,6 +87,14 @@ To become a TSC member:
 
 A Committer is invited to become a TSC member by existing TSC members. A nomination will result in discussion and then a decision by the TSC.
 
+#### Process for Adding TSC Members
+
+1. Add the GitHub user to the "ESLint TSC" team
+2. Set the GitHub user to be have the "Owner" role for the ESLint organization
+3. Send welcome email with link to maintainer guide
+4. Make TSC member an admin on the ESLint team mailing list
+5. Tweet congratulations to the new TSC member from the ESLint Twitter account
```
Comment (platinumazure) [inline docs/maintainer-guide/governance.md]: Don't know if we want to mention updating the README as well?

File: docs/maintainer-guide/governance.md (hunk 1 above)
Comment (alberto) [inline docs/maintainer-guide/governance.md]: Add to gitter room and README

File: docs/maintainer-guide/governance.md (hunk 2 above)
Comment (alberto) [inline docs/maintainer-guide/governance.md]: Add to gitter room, README, tweetdeck, jenkins (anything else?)

File: docs/maintainer-guide/governance.md [hunk 3]
```diff
@@ -81,6 +89,17 @@ To become a TSC member:
...
+#### Process for Adding TSC Members
+
+1. Add the GitHub user to the "ESLint TSC" team
+2. Set the GitHub user to be have the "Owner" role for the ESLint organization
+3. Send welcome email with link to maintainer guide
+4. Add TSC member to the README
+5. Invite to Gitter TSC chatroom
+6. Make TSC member an admin on the ESLint team mailing list
+7. Add TSC member as an admin to ESLint Twitter Account on Tweetdeck
+8. Tweet congratulations to the new TSC member from the ESLint Twitter account
```
Comment (alberto) [inline docs/maintainer-guide/governance.md]: Not sure if you forgot about tweetdeck permissions.

Also, add new member timezone to the next TSC meeting issue. I find it super useful (I always forget the exact time of the meeting :))

Comment (nzakas) [inline docs/maintainer-guide/governance.md]: I say to add them as an admin on tweetdeck, is there some other permission you're referring to?

Comment (alberto) [inline docs/maintainer-guide/governance.md]: oh, sorry. I didn't see that line :/

**Pull request merged**